Git Formats/tests/syntax_test_git_diff_combined.patch

```diff
@@ -0,0 +1,50 @@
+\ SYNTAX TEST "Packages/Git Formats/Git Diff.sublime-syntax"
+diff --combined describe.c
+index fabadb8,cc95eb0..4866510 644
```

Comment:
In Git Log, the `..` is marked as `punctuation.sequence`. I have it here as `keyword.operator.logical`. I can make a case either way, but feel free to express an opinion. (For these "combined" diffs, the `,`  is marked as `punctuation`.)

Reply:
Today, I'd probably use `punctuation.separator.range.diff` as those denote a list of arbitrary commits starting with `cc95eb0` and finalized by `4866510`.

It would match to recent changes made to line range markers in (`diff-line-ranges`).

Reply:
This sounds good. I think `..` and `...` are technically set operators, but people tend to think of them as ranges.

Reply:
Like a python set `{a, b, c}`? Not sure I get it.

Reply:
I was thinking more like `in`.